sdk/core/Azure.Core/perf/DownloadTest.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using Azure.Identity;
+using Azure.Test.Perf;
+
+namespace Azure.Template.Perf
+{
+    public class DownloadTest : PerfTest<DownloadTestOptions>
+    {
+        // Stream.CopyToAsync default buffer
+        private const int BufferSize = 81920;
+        private readonly HttpPipeline _pipeline;
```

Comment:
Static or one per instance?

Reply:
I've seen clients created per-instance in other tests so mimicked it.